website/src/hooks/hubSpot.ts

```diff
@@ -136,20 +157,36 @@ function createHubSpotForm({ region, portalId, formId, targetId, onFormSubmit, o
     })
 }
 
-export const useHubSpot = (
-    region: string,
-    portalId: string,
-    formId: string,
-    targetId: string,
+interface HookProps {
```

Comment:
I usually like keeping relevant code close by since this interface is only used directly after its declared in the hook. But happy to move it up if you prefer thinking of them like hoisted declarations.

Also, I have this code synced up with sourcegraph/about-replatform#53 😄

Reply:
Awesome, yeah. I find it easier to reason about things when interfaces are grouped in one place, personally.

Sweet, I will take a look!